synapse/crypto/context_factory.py

```diff
@@ -32,8 +32,11 @@ def __init__(self, config):
     @staticmethod
     def configure_context(context, config):
         try:
-            _ecCurve = _OpenSSLECCurve(_defaultCurveName)
-            _ecCurve.addECKeyToContext(context)
+            # This was removed in https://github.com/twisted/twisted/pull/928
+            # _ecCurve = _OpenSSLECCurve()
+            _evCurve = crypto.get_elliptic_curve(_defaultCurveName)
```

Comment:
why is it called _evCurve now?

Reply:
Brain corrected EC to EV (work product is called EV, my bad)